src/hooks/usePopulateTableRows.tsx

```diff
@@ -81,6 +82,20 @@ export const usePopulateTableRows = (
     );
   };
 
+  const donationDate = (
+    dateField: 'firstDonationDate' | 'lastDonationDate',
+  ): RenderCell => {
+    const DonationDate: RenderCell = ({ row }) => (
```

Comment:
Since this isn't a React component, just a regular function, it would be good to not capitalize it.

```suggestion
    const donationDate: RenderCell = ({ row }) => (
```